packages/early-boot-config/early-boot-config.spec

```diff
@@ -72,8 +98,27 @@ Requires: %{name}-local
 %endif
     %{nil}
 
-%global early_boot_config_bindir %{_cross_libexecdir}/early-boot-config/bin
-%global early_boot_config_provider_dir %{_cross_libexecdir}/early-boot-config/data-providers.d
+# Store the output so we can print it after waiting for the backgrounded job.
```

Comment:
Why do this for `early-boot-config` but not for `netdog`? Does it make a big difference in build times?